src/Styleguide/Pages/Artist/Routes/Overview/index.tsx

```diff
@@ -93,7 +94,7 @@ export const Overview = () => {
 
       <Row>
         <Col>
-          <ArtworkFilter />
+          <ArtworkFilterQueryRenderer artistID="pablo-picasso" />
```

Comment:
Static page is slowly becoming picasso's page 🎉

Reply:
<img width="245" alt="screen shot 2018-06-23 at 1 05 20 pm" src="https://user-images.githubusercontent.com/236943/41813212-276eb202-76e6-11e8-82c6-0396f45e8e26.png">